### PR TITLE
Not found message on invalid search

### PR DESCRIPTION
**Pull request by Kunal-2001**
Not found display message on invalid search ( algorithms not available).

### CODE REVIEW

File: pages/search/index.tsx [hunk 1]
```diff
@@ -29,7 +29,9 @@ export default function Search() {
     <>
       <Head title={router.query.q && `"${router.query.q}"`} />
       <Section
-        title={`${t("search")}${router.query.q && ` "${router.query.q}"`}`}
+        title={`${algorithms.length === 0 ? t("No results for") : t("search")}${
```
Comment (maltejur) [inline pages/search/index.tsx]: You would still need to add a new string for it to be able to be translated properly. Replace the `t("No results for")` with `t("noResults")` and add `No results for` as a new string here:
https://github.com/TheAlgorithms/website/blob/main/public/locales/en/common.json

**Review by maltejur — CHANGES_REQUESTED** [review]
Nice small change. Maybe also add something for when the query is empty?

File: pages/search/index.tsx (hunk 1 above)
Comment (Kunal-2001) [reply pages/search/index.tsx]: I have made the changes and added a message for empty query, please review.

**Review by maltejur — APPROVED** [review]
Looking good

**Pull request merged**